tests/testMisc.py

```diff
@@ -364,6 +364,28 @@ def test_remap(self):
         assert drawBot.remap(25, 10, 20, 30, 50) == 60
         assert drawBot.remap(25, 10, 20, 30, 50, clamp=True) == 50
 
+    def test_validateLanguageCode(self):
+        expectedLanguageValidation = [
+            ("ab", False),
+            ("abk", False),
+            ("abz", False),
+            ("AB", False),
```

Comment:
Some actual ISO 639-1 vs ISO 639-2 tests:
```suggestion
            ("AB", False),
            ("af", True),
            ("afr", True),
            ("af_NA", True),
            ("afr_NA", True),
            ("af_ZZ", False),
            ("afr_ZZ", False),
```